lib/crawler_web/controllers/user_auth.ex

```diff
@@ -55,6 +59,23 @@ defmodule CrawlerWeb.UserAuth do
     end
   end
 
+  @doc """
+  Used for routes that require the user to be authenticated.
+  If you want to enforce the user email is confirmed before
+  they use the application at all, here would be a good place.
+  """
+  def require_authenticated_user(conn, _opts) do
+    if conn.assigns[:current_user] do
+      conn
+    else
+      conn
+      |> put_flash(:error, "You must log in to access this page.")
```

Comment:
```suggestion
      |> put_flash(:error, gettex("You must log in to access this page."))
```

This is not a part of the IC's requirement but you can have a look on the Localization topic, so you will know about that part @vnntsu just a suggestion 👌 

https://www.wbotelhos.com/i18n-with-phoenix